### DISCUSSION TITLE
能否支持脚本管理器Stay？

### DISCUSSION THREAD

**Discussion by Twilight-9** [the1812/Bilibili-Evolved#4494] [功能建议 / Ideas]
Mac Safari上的Tampermonkey旧不更新，很多时候会自己无法使用。于是转到Stay。但是发现自定义顶栏是一片空白，可能是兼容性问题。

**Comment by RingingVolcano** [comment]
建议换用Chromium系的浏览器吧 官方在浏览器兼容性里也删除了Safari
https://github.com/the1812/Bilibili-Evolved#%E6%B5%8F%E8%A7%88%E5%99%A8
